src/main/java/it/eng/idsa/businesslogic/processor/sender/websocket/client/MessageWebSocketOverHttpSender.java

```diff
@@ -30,27 +28,32 @@
 import it.eng.idsa.multipart.builder.MultipartMessageBuilder;
 import it.eng.idsa.multipart.domain.MultipartMessage;
 import it.eng.idsa.multipart.processor.MultipartMessageProcessor;
+import it.eng.idsa.businesslogic.service.impl.TLSProvider;
 
 /**
- * @author Antonio Scatoloni
+ * Author: Antonio Scatoloni
  */
 
 @Component
 public class MessageWebSocketOverHttpSender {
     private static final Logger logger = LoggerFactory.getLogger(MessageWebSocketOverHttpSender.class);
 
-//    private WebSocketClientConfiguration webSocketClientConfiguration;
     private RejectionMessageService rejectionMessageService;
     private FileStreamingBean fileStreamingBean;
     private ResponseMessageBufferClient responseMessageBufferClient;
     private InputStreamSocketListenerClient inputStreamSocketListenerWebSocketClient;
     
+    @Autowired
+    private TLSProvider tlsProvider;
```

Comment:
Can you remove annotation since it is set via constructor?

Reply:
Good catch, I forgot to delete it when I refactored it.